<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1165</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="ip">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>350</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.0.55</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connect">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>60</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>connect</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="port">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>502</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="timeout">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>35</y>
      <width>195</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>3000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>65</y>
      <width>100</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>disconnect</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="parsetest_tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>350</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="apply">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>145</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>apply</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="apply_test">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>350</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="start_addr">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>115</y>
      <width>350</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>11107,11351,11367,11375,11153</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lenth">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>145</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>12</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>145</y>
      <width>95</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="signLog">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>350</width>
      <height>300</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>780</width>
      <height>621</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_v">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>640</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Reg 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_a">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>640</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Reg 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_kw">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>640</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Reg 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_kwh">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>640</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Reg 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_temp">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>640</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Reg 5</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1165</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
